### PR TITLE
Simplify/Revert backplaneStatus as an attempt to correct broken backplane metrics

### PR DESCRIPTION
**Pull request by 80degreeswest**
Currently, the backplane metrics are broken. Available workers, number of dispatched, queued and prequeued operations are incorrect. This presents a very significant issue as those metrics are used for the primary monitoring and scaling of the entire system. This PR simplifies the metrics and removes additional functionality added between v1.7.0 and v1.9.0. Those additional metrics will need to be re-introduced with correct implementation.

### CODE REVIEW

**Review by luxe — COMMENTED** [review]
I'm good with deleting as much as you need to help rule things out.  For the CI, deleting code will cause more dead code, so you will need to make static analysis happy in that regard.

File: src/main/java/build/buildfarm/instance/shard/RedisShardBackplane.java [hunk 1]
```diff
@@ -1444,212 +1436,14 @@ public boolean canPrequeue() throws IOException {
...
-        .addAllPlatformProperties(toLabeledCounts(platformProperties))
-        .setUniqueClientsAmount(uniqueToolInvocationIds.size())
-        .build();
+    BackplaneStatus.Builder builder = BackplaneStatus.newBuilder();
+    builder.addAllActiveWorkers(client.call(jedis -> jedis.hkeys(config.getWorkersHashName())));
+    builder.setDispatchedSize(
+        client.call(jedis -> jedis.hlen(config.getDispatchedOperationsHashName())));
+    builder.setOperationQueue(operationQueue.status(client.call(jedis -> jedis)));
+    builder.setPreQueueSize(
+        client.call(jedis -> jedis.hlen(config.getPreQueuedOperationsListName())));
```
[the remark this answers is not in the document]
Comment (80degreeswest) [reply src/main/java/build/buildfarm/instance/shard/RedisShardBackplane.java:1445]: Ah ok I'll undo the prequeue change. Didn't realize prequeue was a balanced queue now too.

File: src/main/java/build/buildfarm/instance/shard/RedisShardBackplane.java [hunk 2]
```diff
@@ -579,13 +578,6 @@ static CasWorkerMap createCasWorkerMap(RedissonClient client, RedisShardBackplan
 
   static RedissonClient createRedissonClient(RedisShardBackplaneConfig config) throws IOException {
     Config redissonConfig = new Config();
-
-    ClusterServersConfig finalConfig =
-        redissonConfig
-            .useClusterServers()
-            .addNodeAddress(config.getRedisUri())
-            .setCheckSlotsCoverage(false);
-
```
[the remark this answers is not in the document]
Comment (80degreeswest) [reply src/main/java/build/buildfarm/instance/shard/RedisShardBackplane.java:588]: That config is defined but not used. Is that intentional?

[the remark this answers is not in the document]
Comment (luxe) [reply src/main/java/build/buildfarm/instance/shard/RedisShardBackplane.java:588]: the values are still changed and used to create the client.  
I don't want to block on this, so I'll revisit all the removed code later

**Review by luxe — APPROVED** [review]
alright, let's see!

**Pull request merged**